_episodes/05-starting-jekyll.md

```diff
@@ -7,75 +7,351 @@ questions:
 - "How can I configure global values/settings for my site?"
 - "How can I reuse the same chunks of material in multiple pages?"
 objectives:
-- "substitute variable values into their page content"
-- "adjust the configuration of their pages"
-- "create reusable blocks of content and insert them into pages"
+- "Substitute variable values into a page content"
+- "Adjust the configuration of site and individual pages"
+- "Create reusable blocks of content and insert them into pages"
 keypoints:
-- "Variables can be defined globally in `_config.yml` or locally in the YAML header"
-- "Variable values can be substituted into page content with `{{ double_curly_brackets }}`"
+- "Variables can be defined globally in `_config.yml` or locally within YAML header (*front matter*) of a page"
+- "Variable values can be substituted into page content with Liquid notation: `{{ variable }}`"
+- "Global variables are accessible from any page of your website; local variables can only be accessed within 
+the page they were defined in (and any pages that include this page)"
 ---
 
-- create and set a parameter in YAML header ("front matter")
-- use parameter in page body
-- use a global parameter from `_config.yml`
-- use it again in another page
-- change the value of the global parameter and study the effect
+[Jekyll](https://jekyllrb.com/) is a powerful static site generator behind GitHub Pages. It creates static HTML
+website content out of various files in your repository (Markdown files, CSS style sheets, page templates/layouts, etc.).
+This 'compiled' content is then served as you website via `github.io` Web domain (remember your website's URL
+from the previous episode?). Jekyll automatically re-generates all the HTML pages for your website each time you 
+commit a change to your repository.
```

Comment:
```suggestion
make a change to your repository.
```
I'm fine with commit too but thought that using make might be more accessible for folks who don't know cmdline git.